src/main/app/pay/payClient.ts

```diff
@@ -104,23 +116,29 @@ export class MockPayClient implements PayClient {
       })
   }
 
+  async update (user: User, paymentReference: string, caseReference: string, caseNumber: string): Promise<void> {
+    const payUpdateDelayInMs = 681
+    await delay(payUpdateDelayInMs)
+
```

Comment:
i am not sure if above 2 lines are necessary..

Reply:
Why the delay?

Reply:
I have seen in other mock implementation too, they mimic certain milli sec of delay to mock delay the delay in response. I did the same.